tests/functional/test_status.py

```diff
@@ -0,0 +1,10 @@
+class TestStatus:
+    def test_it(self, app):
+        response = app.get("/_status")
+
+        assert response.content_type == "application/json"
+        assert response.json == {"status": "okay"}
+        assert (
+            response.headers["Cache-Control"]
+            == "max-age=0, must-revalidate, no-cache, no-store"
```

Comment:
Yeah I don't know that this makes sense as a combination of values strictly but it's what Pyramid generates.